package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "rururunbook",
+  "description": "rururunbook is a CLI tool that automates security incident response",
+  "version": "1.0.0",
+  "files": [
+    "lib"
+  ],
+  "bin": {
+    "runbook": "lib/index.js"
+  },
+  "dependencies": {
+    "@types/node": "^12.7.5",
```

Comment:
I think ts-node and @types/node can be devDeps. Should make the package a lot smaller at distribution time :)